source/chapters/appendix.rst

```diff
@@ -17,13 +17,7 @@ There are a variety of other resources where you can find out more:
 
 * **Mixxx Wiki**: For up-to-date information about supported DJ hardware,
   documentation on making controller mappings, and other tips and tricks, go to:
-  `<https://github.com/mixxxdj/mixxx/wiki>`_
-
-* **Keyboard Shortcuts**: This a list for all default Mixxx keyboard shortcuts.
-  Please note that these shortcuts are for keyboards configured in English
-  layout and default shortcuts may be different if your language setting
-  is not English.
-  :ref:`default keyboard mappings<appendix-keyboard>`
+  `<https://mixxx.org/wiki/>`_
```

Comment:
please revert that.
Even though we have rediection in pace it's better to have the correct links here.

Reply:
ok!